x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/file_data_visualizer_view/file_data_visualizer_view.js

```diff
@@ -369,38 +344,17 @@ export class FileDataVisualizerView extends Component {
               fileContents={fileContents}
               data={data}
               dataViewsContract={this.props.dataViewsContract}
-              showBottomBar={this.showBottomBar}
-              hideBottomBar={this.hideBottomBar}
               fileUpload={this.props.fileUpload}
               getAdditionalLinks={this.props.getAdditionalLinks}
               capabilities={this.props.capabilities}
+              mode={MODE.IMPORT}
```

Comment:
❓ shall we pass the `mode` variable instead of the constant?

Reply:
Updated in https://github.com/elastic/kibana/pull/159722/commits/5e1635688ad26e348f14f8d7b5a8dbafb0119804